app/assets/javascripts/init.js

```diff
@@ -0,0 +1,272 @@
+
+		if(!$letter.is(':last-child')) {
+		 	setTimeout(function(){ hideLetter($letter.next(), $word, $bool, $duration); }, $duration);
+		} else if($bool) {
+		 	setTimeout(function(){ hideWord(takeNext($word)) }, animationDelay);
+		}
+
+		if($letter.is(':last-child') && $('html').hasClass('no-csstransitions')) {
+			var nextWord = takeNext($word);
+			switchWord($word, nextWord);
+		}
+	}
+
+	function showLetter($letter, $word, $bool, $duration) {
+		$letter.addClass('in').removeClass('out');
+
+		if(!$letter.is(':last-child')) {
+			setTimeout(function(){ showLetter($letter.next(), $word, $bool, $duration); }, $duration);
+		} else {
+			if($word.parents('.cd-headline').hasClass('type')) { setTimeout(function(){ $word.parents('.cd-words-wrapper').addClass('waiting'); }, 200);}
+			if(!$bool) { setTimeout(function(){ hideWord($word) }, animationDelay) }
+		}
+	}
+
+	function takeNext($word) {
+		return (!$word.is(':last-child')) ? $word.next() : $word.parent().children().eq(0);
+	}
+
+	function takePrev($word) {
+		return (!$word.is(':first-child')) ? $word.prev() : $word.parent().children().last();
+	}
+
+	function switchWord($oldWord, $newWord) {
+		$oldWord.removeClass('is-visible').addClass('is-hidden');
+		$newWord.removeClass('is-hidden').addClass('is-visible');
+	}
+
+	$('.button-collapse').sideNav({
+		menuWidth: 240, // Default is 240
+		closeOnClick: true // Closes side-nav on <a> clicks, useful for Angular/Meteor
+	});
+
+	$('.parallax').parallax();
+
+	var card  = document.querySelectorAll('.card-work');
+  	var transEndEventNames = {
+	      'WebkitTransition' : 'webkitTransitionEnd',
+	      'MozTransition'    : 'transitionend',
+	      'transition'       : 'transitionend'
+	},
+	transEndEventName = transEndEventNames[ Modernizr.prefixed('transition') ];
+
+	function addDashes(name) {
+		return name.replace(/([A-Z])/g, function(str,m1){ return '-' + m1.toLowerCase(); });
+	}
+
+	function getPopup(id) {
+		return document.querySelector('.popup[data-popup="' + id + '"]');
+	}
+
+	function getDimensions(el) {
+	   return el.getBoundingClientRect();
+	}
+
+	function getDifference(card, popup) {
+		var cardDimensions = getDimensions(card),
+	    	popupDimensions = getDimensions(popup);
+
+		return {
+		  	height: popupDimensions.height / cardDimensions.height,
+		  	width: popupDimensions.width / cardDimensions.width,
+		  	left: popupDimensions.left - cardDimensions.left,
+		  	top: popupDimensions.top - cardDimensions.top
+		}
+	}
+
+	function transformCard(card, size) {
+		return card.style[Modernizr.prefixed('transform')] = 'translate(' + size.left + 'px,' + size.top + 'px)' + ' scale(' + size.width + ',' + size.height + ')';
+	}
+
+	function hasClass(elem, cls) {
+	    var str = " " + elem.className + " ";
+	    var testCls = " " + cls + " ";
+	    return(str.indexOf(testCls) != -1) ;
+	}
+
+	function closest(e) {
+	   var el = e.target || e.srcElement;
+	    if (el = el.parentNode) do { //its an inverse loop
```

Comment:
Expected a conditional expression and instead saw an assignment.<br>Expected '{' and instead saw 'do'.